source/addComment.js

```diff
@@ -1,3 +1,54 @@
 $(document).ready(function(){
-  // add your code here
+  $('form').on('submit', function(event) {
+    event.preventDefault()
+  })
+  var newCommentButton = document.getElementById('new_comment_button')
+  var submitButton = document.getElementsByTagName('input')[1]
+
+  newCommentButton.addEventListener('click', renderCommentForm)
+  submitButton.addEventListener('click', postComment)
```

Comment:
Another thing you could have done was called `postComment()` in the above `$('form').on('submit',......`